spec/client/controllers/MainController.spec.js

```diff
@@ -139,18 +139,19 @@ describe("MainController", function () {
         it("adds special html tag for displaying hashtags inside tweets", function () {
             expect(tweetTextManipulationService.addHashtag("#hello world", [{
                 text: "hello"
-            }])).toEqual(" <b>#hello</b>  world");
+            }])).toEqual("<b>#hello</b> world");
```

Comment:
Although you've fixed these tests, I think they should actually just be taken out altogether; they're exactly the same as the ones being run in the tweetTextManipulationService spec, which is where they should be